File: Source/UserManagement/Domain/DataCollector/Changing/ChangePreferredLanguage.cs
using System;
using Concepts;
using Concepts.DataCollector;
using Dolittle.Commands;

namespace Domain.DataCollector.Changing
{
    public class ChangePreferredLanguage : ICommand
    {
        public DataCollectorId DataCollectorId { get; set; }
        public Language PreferredLanguage { get; set; }
    }
}

File: Source/UserManagement/Web.Angular/src/app/domain/data-collector/ChangePreferredLanguage.ts
import { Command } from '../../services/Command';
import { Language } from '../language.model';

export class ChangePreferredLanguage extends Command {
    dataCollectorId: string;
    preferredLanguage: number

    constructor() {
        super();
        this.type = 'CBS#UserManagement.DataCollector.Changing-ChangePreferredLanguage+Command@1';
    }
}
